.github/workflows/GnuTests.yml

```diff
@@ -137,12 +138,32 @@ jobs:
         path_GNU='gnu'
         path_UUTILS='uutils'
         bash "uutils/util/run-gnu-test.sh" run-root
+        ### Run tests as root
+
     - name: Extract testing info from individual logs (run as root) into JSON
       shell: bash
       run : |
         path_UUTILS='uutils'
         python uutils/util/gnu-json-result.py gnu/tests > ${{ env.TEST_ROOT_FULL_SUMMARY_FILE }}
 
+    - name: Run GNU stty tests
```

Comment:
please add a comment explaining why it needs it own section

Reply:
Added an explanation